src/components/Room/RoomAudioControls.tsx

```diff
@@ -67,7 +82,7 @@ export function RoomAudioControls({ peerRoom }: RoomAudioControlsProps) {
         title={
           isSpeakingToRoom
             ? 'Turn off microphone'
-            : 'Turn on microphone and speak to room'
+            : 'Turn on microphone and speak to room [Hold Ctrl + D to speak]'
```

Comment:
I'd suggest tweaking this label a bit so it flows a bit more like nontechnical text.
```suggestion
            : 'Turn on microphone and speak to room (hold ctrl + D to speak)'
```